js/main.js

```diff
@@ -0,0 +1,345 @@
+        <h2 class="heading">${projects[i].title}</h2>
+        <ul class="features">
+          <li class="feature">${projects[i].features[0]}</li>
+          <li class="feature">${projects[i].features[1]}</li>
+          <li class="feature">${projects[i].features[2]}</li>
+        </ul>
+        <button data-id="${projects[i].id}" class="c-projects">
+          See this project &rarr;
+        </button>
+      </div>
+    </li>
+  `;
+
+    projectList.innerHTML += projectItemtemplate;
+  }
+}
+
+window.addEventListener('load', loadProjects());
+
+function createModal(id) {
+  const pCardTemplate = `
+  <div class="overlay hidden">
+      <article class="modal container">
+        <div class="modal__header">
+          <div class="modal__header_title flex space-between">
+            <h1 class="top_title">${projects[id].title}</h1>
+            <button type="button" class="top_btn">
+              &#x2716
+            </button>
+          </div>
+          <ul role="list" class="modal__header_bottom features-container flex">
+            <li class="feature">${projects[id].features[0]}</li>
+            <li class="feature">${projects[id].features[1]}</li>
+            <li class="feature">${projects[id].features[2]}</li>
+          </ul>
+        </div>
+        <div class="modal__body">
+          <div class="modal__body_slide">
+            <div class="slide_main">
+              <span class="slide_prev flex">&lt;</span>
+              <div class="slide_show">
+                <img src="${projects[id].image}" />
+              </div>
+              <span class="slide_next flex">&gt;</span>
+            </div>
+            <ul class="slide_items flex" role="list">
+              <li class="slide_item active">
+                <img src="${projects[id].sideImage[0]}" alt="project" />
+              </li>
+              <li class="slide_item">
+                <img src="${projects[id].sideImage[1]}" alt="project" />
+              </li>
+              <li class="slide_item">
+                <img src="${projects[id].sideImage[2]}" alt="project" />
+              </li>
+              <li class="slide_item">
+                <img src="${projects[id].sideImage[3]}" alt="project" />
+              </li>
+            </ul>
+          </div>
+          <div class="modal__body_wrapper">
+            <div class="modal__body_description">
+              <p>${projects[id].description}</p>
+            </div>
+            <div class="modal__body_buttons flex direction">
+              <button class="btn secondary-btn flex" type="button">
+                <span>See Live</span>
+                <img src="./assets/icons/arw.png" alt="see this project" />
+              </button>
+              <button class="btn secondary-btn flex" type="button">
+                <span>See Source</span>
+                <img src="./assets/icons/gitb.png" alt="see this project" />
+              </button>
+            </div>
```

Comment:
- [ ] Kindly, fix the "See live" and "See source" buttons, the images are not appearing, and please, also check the size. 🌵 
- [ ] Please. also fix the buttons in the mobile version 🦖.

Your buttons:

![image](https://user-images.githubusercontent.com/91301423/202871523-0a8d63e7-e339-4602-a0b0-693c89e3f020.png)

Figma buttons:

![image](https://user-images.githubusercontent.com/91301423/202871542-9227d97b-a9dc-45c2-9614-77ab163f8f19.png)